nikola/nikola.py

```diff
@@ -1675,6 +1675,8 @@ def generic_rss_feed(self, lang, title, link, description, timeline,
 
         feed_append_query = None
         if rss_links_append_query:
+            if isinstance(rss_links_append_query, bool):
```

Comment:
A better, more obvious check would be `if rss_links_append_query is True:`

Reply:
good call. I'll commit that.